hdr_html_canvas_element.md

```diff
@@ -220,31 +220,46 @@ dictionary CanvasColorMetadata {
 ```
 
 ```idl
-dictionary CanvasMasteringDisplayMetadata {
-  optional ColorVolume colorVolume;
+dictionary CanvasColorVolumeMetadata {
+  optional Chromaticity chromaticity;
   optional double minimumLuminance;
-  optional double maximumLuminance;
+  required double maximumLuminance;
 }
 ```
 
-If present, `masteringDisplayMetadata` specifies the characteristics (the color
-primaries, white point, and luminance range) of the display that was used in
-mastering the image content. This information allows a destination display to
-optimize tone mapping based on the relationship between its own capabilities and
-those of the mastering display. [SMPTE ST
-2086:2018](https://ieeexplore.ieee.org/document/8353899) specifies the semantics
-and range of values permitted:
-* `redPrimaryX`, `redPrimaryY`, `greenPrimaryX`, `greenPrimaryY`,
-  `bluePrimaryX`, and `bluePrimaryY` are the xy coordinates, as defined in [ISO
-  11664-3](https://www.iso.org/standard/74165.html) of the nominal primaries of
-  the mastering display
-* `whitePointX` and `whitePointY` are the xy coordinates of the nominal
-  chromaticity of the white point of the mastering display
-* `minimumLuminance` and `maximumLuminance` are the nominal minimum and maximum
-  display luminance, respectively, of the mastering display in cd/m².
-
-The attributes of masteringDisplayMetadata SHOULD be set if know, e.g. if
-obtained from metadata contained in a source image, and omitted otherwise.
+`colorVolumeMetadata` specifies the nominal color volume occupied by
+the image content in the CIE 1931 XYZ color space. The boundaries of the color
+volume are defined by:
+
+* the xy coordinates, as defined in [ISO
+  11664-3](https://www.iso.org/standard/74165.html), of three color primaries:
+  `redPrimaryX`, `redPrimaryY`, `greenPrimaryX`, `greenPrimaryY`,
+  `bluePrimaryX`, and `bluePrimaryY`;
+* the xy coordinates of a white point: `whitePointX` and `whitePointY`; and
+* a minimum and maximum luminance in cd/m²: `minimumLuminance` and `maximumLuminance`.
+
+If omitted, `chromaticity` is equal to the chromaticity of the color space of
+the Canvas.
+
+If omitted, `minimumLuminance` is equal to 0.
+
+The color volume is nominal because it MAY be smaller or larger than the actual
+color volume of image content, but SHOULD not be larger.
```

Comment:
Do you instead mean
```suggestion
color volume of image content, but SHOULD not be smaller.
```
?

If the nominal color volume is smaller than the actual image content color volume, and color gamut mapping is driven by the nominal volume, then that would result in unexpected color clipping or worse.

Reply:
I agree. If it is smaller, that means the content contains values that were out of gamut for the mastering display. That seems difficult to explain. But certainly, the color volume of a given image may easily be smaller than the gamut of the mastering display (a grayscale or sepia image; an image that uses primarily cool or warm hues, for example).

Reply:
Thanks for catching the typo.